### ISSUE TITLE
Can I get both orignal video and watermark video?

### ISSUE THREAD

**Issue by KunNiu** [natario1/CameraView#1199]
### How do I?
Can I get both orignal video and watermark video?

### Version used
The last version.

**Comment by SweetD3v** [comment]
No..You can't. Because watermark video is snapshot video. Captures everything in the preview screen.
It's not possible to get 2 different videos at a time (With & without watermark). 
So For now, you've to record it separately or record original video & then apply watermark using ffmpeg.
